vuepress/docs/next/tutorials/create/ms/add-access-controls.md

```diff
@@ -1,109 +1,161 @@
+3. Modify the `deleteConference` method by adding the following annotation
 ```
     @PreAuthorize("hasAuthority('conference-admin')")
     public ResponseEntity<Void> deleteConference(@PathVariable Long id) {
 ```
-Here we're restricting the delete method to only the `conference-admin` role.
 
-12. Restart the microservice. By default this will include rebuilding any changed source files.
-13. Once the microservice is available, go back to the MFE and try deleting one of the Conferences in the list. You should be able to attempt the delete in the UI but you'll get a 403 error in the browser console and an error like this in the service logs:
+To verify that a user without the `conference-admin` role is unable to call the delete API:
+
+1. Restart the microservice. By default this includes rebuilding any changed source files.
+2. Once the microservice is available, return to the MFE and try deleting one of the Conferences in the list 
+3. Verify that attempting to delete via the UI generates a `403 error` in the browser console and an error in the service logs similiar to the following:
 ```
 2021-03-22 15:56:16.205  WARN 3208 --- [  XNIO-1 task-3] o.z.problem.spring.common.AdviceTraits   : Forbidden: Access is denied
 ```
-That's exactly what we wanted! This demonstrates that a user without `conference-admin` is unable to call the delete API.
 
-Next, let's update the MFE so a user without the `conference-admin` authority cannot even see the delete button in the UI.
+### Step 8: Hide the delete button
 
-14. Edit the `ConferenceTableContainer.js` under `ui/widgets/conference/tableWidget/src/components`. Replace the onDelete logic with an additional check on the user's authorities.
+The MFE UI can be updated to hide the delete button from a user without the `conference-admin` authority. The key logic checks whether the `internal` client role `conference-admin` is mapped to the current user via the hasResourceRole call.
+
+1. Go to the `ui/widgets/conference/tableWidget/src/components` directory
+2. Open `ConferenceTableContainer.js` 
+3. Replace the `onDelete` logic with an additional user permissions
 ```
     const isAdmin = (keycloak && keycloak.authenticate) ? keycloak.hasResourceRole("conference-admin", "internal"): false;
     const showDelete = onDelete && isAdmin;
 
     const Actions = ({ item }) =>
       showDelete ? (
 ```
+4. Confirm that the delete icon is no longer visible in the MFE, which should have automatically reloaded
 
-The key logic there is the hasResourceRole call which checks whether the `internal` client role `conference-admin` was mapped to the current user.
-
-15. View the MFE (whch should have automatically reloaded) and you should see that the delete icon is no longer visible, matching the admin's current permissions.  We've now verified that a user with just `conference-user` can neither see the delete action in the UI nor call its corresponding API.
+### Step 9: Grant and verify delete permissions
```

Comment:
should Step headers be capitalized ? In some parts they are capped, in others, not.

Reply:
they are not capitalized unless they include proper nouns, in which case that's a coincidence and not a convention